src/DataFixtures/ORM/LoadCommitteeAdherentMandateData.php

```diff
@@ -45,6 +45,7 @@ public function getDependencies()
     {
         return [
             LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`